docs/core/proto-docs.md

```diff
@@ -264,6 +264,32 @@
   
     - [Msg](#cosmos.evidence.v1beta1.Msg)
   
+- [cosmos/fee_grant/v1beta1/fee_grant.proto](#cosmos/fee_grant/v1beta1/fee_grant.proto)
```

Comment:
looks like the docs are not updated after changing the package name

Reply:
This is because of, `make proto-all` command is failing. unable generate `proto-docs.md` automatically.